src/ucp/core/ucp_ep.c

```diff
@@ -2727,6 +2730,14 @@ static void ucp_ep_req_purge(ucp_ep_h ucp_ep, ucp_request_t *req,
         }
 
         ucp_request_put(req);
+    } else if (req->send.uct.func == ucp_rma_sw_proto.progress_get) {
```

Comment:
what about progress_put/atomic?

Reply:
good catch, done
can handle here as well, the only difference is memory deregistration, but md_map is `0` - so, can call it for AMO as well